examples/language/gpt/gemini/commons/utils.py

```diff
@@ -30,8 +30,9 @@ def get_profile_context(enable_flag, warmup_steps, active_steps, save_dir):
             activities=[ProfilerActivity.CPU, ProfilerActivity.CUDA],
             schedule=schedule(wait=0, warmup=warmup_steps, active=active_steps),
             on_trace_ready=tensorboard_trace_handler(save_dir),
-            record_shapes=True,
-            profile_memory=True,
+            # record_shapes=True,
+            # profile_memory=True,
```

Comment:
why not record shapes

Reply:
It seems it's dupilcated as examples/language/performance_evaluator.py

Reply:
> It seems it's dupilcated as examples/language/performance_evaluator.py

Resolved

This was previously only used for gpt and is now moved to `performance_evaluator.py` for llama.